tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -204,7 +204,7 @@ def test_hyperopt_run_hyperopt(csv_filename):
                     "lower": 0.001,
                     "upper": 0.1,
                 },
-                output_feature_name + ".fc_size": {"space": "randint", "lower": 32, "upper": 256},
+                output_feature_name + ".output_size": {"space": "randint", "lower": 32, "upper": 256},
```

Comment:
these values is likely too big, let's decrease it for faster tests

Reply:
Done